### ISSUE TITLE
predictions point number

### ISSUE THREAD

**Issue by gxu12** [CuiRuikai/Partial2Complete#11]
Hi,
really appreciate for your work! is it possible to set the output points larger than 2048 points?

**Comment by CuiRuikai** [comment]
Yes, just change the output dimension of last decoder layer could achieve this
